corehq/apps/locations/fixtures.py

```diff
@@ -219,14 +219,42 @@ def should_sync_flat_fixture(project):
 )
 
 
+class RelatedLocationSerializer(FlatLocationSerializer):
+
+    def get_xml_nodes(self, fixture_id, restore_user, locations_queryset, data_fields):
+        all_types = LocationType.objects.filter(domain=restore_user.domain).values_list(
+            'code', flat=True
+        )
+        location_type_attrs = ['{}_id'.format(t) for t in all_types if t is not None]
+        attrs_to_index = ['@{}'.format(attr) for attr in location_type_attrs]
+        attrs_to_index.extend(_get_indexed_field_name(field.slug) for field in data_fields
+                              if field.index_in_fixture)
```

Comment:
You could bump this stuff into a helper method so it's not duplicated

Reply:
will do this in a follow up